xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -283,10 +306,17 @@ This constructor is protected. It can only be called from the <xref:System.Runti
         <Parameter Name="activating" Type="System.Reflection.Assembly" Index="0" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="activating">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="activating">Assembly which will be used to determine the requested <see cref="T:System.Runtime.Loader.AssemblyLoadContext" /> instance.</param>
+        <summary>Sets <see cref="P:System.Runtime.Loader.AssemblyLoadContext.CurrentContextualReflectionContext" /> to the <see cref="T:System.Runtime.Loader.AssemblyLoadContext" /> which loaded activating.</summary>
```

Comment:
which loaded activating?

Reply:
I was trying to refer to the parameter name `activating`.  Changed to `which loaded the assembly.`